backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/TriggerBuilderHelpers.java

```diff
@@ -0,0 +1,38 @@
+package org.molgenis.emx2.sql;
+
+public class TriggerBuilderHelpers {
+
+  private TriggerBuilderHelpers() {
+    throw new IllegalStateException("Utility class");
+  }
+
+  public static String buildOntologySearchSection(
+      String schemaName, String tableName, String columnName, boolean isArray) {
+    String template =
+        isArray
+            ? """
```

Comment:
Evil code ;-) you could have used jooq DSL builder for this. I don't mind this, jooq is evil in its own way.
So this comment is just FYI. That would be something like DSLContext.select() etc.

Reply:
100 % agree , can you suggest a DSL version for this ?